openedx/features/edly/models.py

```diff
@@ -0,0 +1,37 @@
+from django.contrib.sites.models import Site
+from django.core.validators import RegexValidator
+from django.db import models
+from model_utils.models import TimeStampedModel
+from organizations.models import Organization
+
+ALPHANUMERIC_VALIDATOR = RegexValidator(r'^[0-9a-zA-Z]*$', 'Only alphanumeric characters are allowed.')
```

Comment:
As discussed here https://edlyinc.slack.com/archives/C010MM9C282/p1588660720005700 we need to update this to just lowercase aphanumeric characters and hyphens (https://docs.djangoproject.com/en/3.0/ref/utils/#django.utils.text.slugify) without unicodes.

@ziafazal @asadmanzoor93 My suggestion would be to remove this validation altogether and just automatically slugify the value provided in the name.

https://github.com/edly-io/edly-panel-backend/blob/master/edly_panel/apps/organizations/models.py#L29-L34

We should have following pattern for single customer single site:
```
{edx-org-short_name}.edly.io
{edx-org-short_name}.courses.edly.io
{edx-org-short_name}.studio.edly.io
```
and for single customer multi site:
```
{edly-org-slug}.{edx-org-short_name}.edly.io
{edly-org-slug}.{edx-org-short_name}.courses.edly.io
{edly-org-slug}.{edx-org-short_name}.studio.edly.io
```
Note that the `slug` and `short_name` can have hyphen `-` other then alphanumeric values. This way will be able to split domain name on dot `.` and get the `slug` and `short_name` based on single site customer URL or multi-site customer URL.

Reply:
@ziafazal this way the slug in the URL will be automatically generated. Do we want to allow our cloud customers to chose the `edly-org` and `edly-sub-org` slugs?

Reply:
@zubair-arbi yeah we do want our cloud customers to choose their slug.

Reply:
@asadmanzoor93 rename `ALPHANUMERIC_VALIDATOR` to `EDLY_SLUG_VALIDATOR`

Reply:
Done